README.md

```diff
@@ -19,13 +21,6 @@ Speak to us on our Community Slack channel (Register at https://launchpass.com/l
 
 A series of next-generation Connectors are in active development. Give us your feedback of which connectors we should be working on or to to get the latest information, send us an email at info@lenses.io
 
-# Stream Reactor Kafka Connectors
-
-![Alt text](images/streamreactor-logo.png)
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Flensesio%2Fstream-reactor.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Flensesio%2Fstream-reactor?ref=badge_shield)
-
-A collection of components to build a real time ingestion pipeline.
-
 ## Kafka Compatibility
 
 * Kafka 2.8 -> 3.5 (Confluent 6.2 -> 7.5) - Stream Reactor 4.1.0+
```

Comment:
whilst you're updating this, we can boost the confluent and kafka versions:
    val kafkaVersion:     String = "3.7.0"
    val confluentVersion: String = "7.6.1"